app/bundles/CoreBundle/IpLookup/IpstackLookup.php

```diff
@@ -30,7 +30,7 @@ protected function getUrl()
             $this->logger->warning('FreeGeoIP has become IPStack and now requires an API key.');
         }
 
-        return 'http://api.ipstack.com/'.$this->ip.'?access_key='.$this->auth.'&output=json&legacy=1';
+        return 'http://api.ipstack.com/' . $this->ip . '?access_key=' . $this->auth . '&output=json&legacy=1';
```

Comment:
Please revert this change back as it breaks code style rules.
```suggestion
        return 'http://api.ipstack.com/'.$this->ip.'?access_key='.$this->auth.'&output=json&legacy=1';
```